www/include/configuration/configObject/contact/formContact.php

```diff
@@ -344,7 +354,12 @@
 /**
  * Contact Groups Field
  */
-$form->addElement('header', 'groupLinks', _("Group Relations"));
+if ($platformType === PLATFORM_CLOUD) {
+    $form->addElement('header', 'groupLinks', _("Roles Relation"));
```

Comment:
missing translation too
```suggestion
    $form->addElement('header', 'groupLinks', _("Role Relations"));
```